buf.gen.yaml

```diff
@@ -0,0 +1,15 @@
+version: "v1"
+managed:
+  enabled: true
+  go_package_prefix:
+    default: "github.com/authzed/api/proto"
+    except:
+      - "buf.build/googleapis/googleapis"
+      - "buf.build/envoyproxy/protoc-gen-validate"
+      - "buf.build/grpc-ecosystem/grpc-gateway"
```

Comment:
Why are these needed?

Reply:
Oof you're correct, they are not needed! I actually don't need the `go_prefix` setting because the protos define them.